app/jobs/reports/combined_invoice_supplement_report_v2.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Reports
+  class CombinedInvoiceSupplementReportV2 < BaseReport
+    REPORT_NAME = 'combined-invoice-supplement-report-v2'
```

Comment:
one more super important thing, we need to add this to `config/initializers/job_configurations.rb` otherwise the job will not run

And depending on how it goes, since this is running basically table scans on `sp_return_logs` we might want to make sure it runs *after* the reports v1 so we could schedule it to run an hour later? ex 1-2am UTC instead of midnight?

Reply:
Done!